packages/node-core/src/utils/graphql.ts

```diff
@@ -112,5 +114,70 @@ export function getColumnOption(
       }
     };
   }
+  if (field.jsonInterface) {
+    columnOption.get = function () {
+      const dataValue = this.getDataValue(field.name);
+
+      if (!dataValue || !field.jsonInterface) {
+        return field.isArray ? [] : null;
+      }
+      return field.isArray
+        ? dataValue.map((v: any) => processNestedJson(field, v, 'Get'))
+        : processNestedJson(field, dataValue, 'Get');
+    };
+    columnOption.set = function (val: unknown) {
+      if (val === undefined || isNull(val)) {
+        this.setDataValue(field.name, null);
+        return;
+      }
+      if (isArray(val)) {
+        const setValue = val.length === 0 ? [] : val.map((v) => processNestedJson(field, v, 'Set'));
+        this.setDataValue(field.name, setValue);
+      } else if (isObject(val)) {
+        this.setDataValue(field.name, processNestedJson(field, val, 'Set'));
+      } else {
+        throw new Error(`input for Json type is only support object or array, got type ${typeof val}`);
+      }
+    };
+  }
   return columnOption;
 }
+
+type OperationType = 'Get' | 'Set';
+
+/***
+ * Process nested json set/get, output is same as input value type
+ * @param field
+ * @param value
+ * @param operationType
+ */
+function processNestedJson(
```

Comment:
I don't think this is backwards compatible. Currently bigInts will have the `n` suffix (or hex for the network project) which doesnt work.

A problem i see with this is that there is no requirement that value has the same structure as the fields. An alternative would be to use the [reviver on JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#using_the_reviver_parameter) to convert BigInts using the `n` suffix.

The downside to this is that the user might want a string with a digits followed by an `n` would be converted to a BigInt when it shouldn't. But it would also be backwards compatible with the special case used by the network project because it wouldn't convert hex. 

I think this function could also be further generalised. What happens if we want to do the same with Dates? I think the same issue would exist